### PR TITLE
[AIRFLOW-6997] Fix worker pods initcontainers

### PR DESCRIPTION
**Pull request by krisgeus**
…rom config

---
Issue link: [AIRFLOW-6997](https://issues.apache.org/jira/browse/AIRFLOW-6997)

Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by Fokko — APPROVED** [review]
LGTM

@dimberman Any final thoughts?

File: airflow/contrib/kubernetes/worker_configuration.py [hunk 1]
```diff
@@ -73,6 +73,13 @@ def _get_init_containers(self):
             'name': 'GIT_SYNC_REV',
             'value': self.kube_config.git_sync_rev
         }]
+
+        for env_var_name, env_var_val in six.iteritems(self.kube_config.kube_env_vars):
```
Comment (BasPH) [inline airflow/contrib/kubernetes/worker_configuration.py:77]:
```suggestion
        for env_var_name, env_var_val in self.kube_config.kube_env_vars.items():
```

Airflow dropped Python 2 support, so no need to use the six library. Other than that, looks good to me.

Comment (Fokko) [reply airflow/contrib/kubernetes/worker_configuration.py:77]: This is targeted for the v1-10 branch

**Comment by dimberman** [comment]
@Fokko lgtm!

**Comment by Fokko** [comment]
Thanks @krisgeus for the PR 👍

**Pull request merged**